frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMerge.java

```diff
@@ -0,0 +1,46 @@
+package com.virtuslab.gitmachete.frontend.actions.base;
+
+import static com.virtuslab.gitmachete.frontend.actions.backgroundables.FetchBackgroundable.LOCAL_REPOSITORY_NAME;
+import static com.virtuslab.gitmachete.frontend.actions.common.ActionUtils.createRefspec;
+import static com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.format;
+import static com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString;
+
+import com.intellij.openapi.project.Project;
+import git4idea.repo.GitRepository;
+import lombok.val;
+
+import com.virtuslab.gitmachete.frontend.actions.backgroundables.FetchBackgroundable;
+import com.virtuslab.gitmachete.frontend.actions.backgroundables.MergeCurrentBranchFastForwardOnlyBackgroundable;
+import com.virtuslab.gitmachete.frontend.actions.common.MergeProps;
+
+public abstract class BaseFastForwardMerge {
```

Comment:
Maybe just final class with private c'tor? 🤔

Reply:
As typically done for such utility classes, also in this project (see e.g. `com.virtuslab.gitmachete.frontend.actions.common.ActionUtils` and many others)